python_modules/dagster/dagster/_cli/workspace/cli_target.py

```diff
@@ -244,7 +270,10 @@ def python_target_click_options():
         click.option(
             "--module-name",
             "-m",
-            help="Specify module where repository or job function lives",
+            multiple=True,
```

Comment:
I think we will want to apply this to package-name as well - they used to be (subtly) different but now are identical. I believe the quickstarts all use the equivalence of package-name currently: https://github.com/dagster-io/dagster-cloud-serverless-quickstart/blob/main/workspace.yaml#L2

And the docs recommend package-name as well and call module-name the deprecated one: https://docs.dagster.io/concepts/repositories-workspaces/workspaces#loading-via-python-package

Reply:
Ah this is a good discussion to have then.

IMO we should actually standardize on module and then deprecated and eliminate package. I had also noticed that the code was identical was identical at this point.

Python's own documentation is inconsistent on this, but the explanation on the import statement page makes the most sense to me given the behavior we see. https://docs.python.org/3/reference/import.html

Pertinent quote:

_"It’s important to keep in mind that all packages are modules, but not all modules are packages. Or put another way, packages are just a special kind of module. Specifically, any module that contains a __path__ attribute is considered a package."_

Given this we can just say that when we say _module_ we mean the target of an import statement.  I think it is both the easiest to explain and the most flexible.